lib/internal/process/pre_execution.js

```diff
@@ -650,6 +653,11 @@ function markBootstrapComplete() {
   internalBinding('performance').markBootstrapComplete();
 }
 
+function getHeapSnapshotFilename(diagnosticDir) {
+  if (!diagnosticDir) return undefined;
+  return `${diagnosticDir}/Heap.${DateNow()}.heapsnapshot`;
```

Comment:
The format here is different from the standard format generated by `DiagnosticFilename()` from the C++ land (the old behavior): https://github.com/nodejs/node/blob/4b610b4b6f05583116b27bddd3cb7a59591f9a6b/src/util.cc#L296

Reply:
I thought because we use a custom directory, we could have a custom naming format. And I was waiting for someone to provide their feedback on it.

Thanks for the suggestion. I have added the same formatting. Please check now.